server/app.py

```diff
@@ -868,7 +868,15 @@ def customer_subscription_created(event):
     # It starts by looking for a matching Contact (or creating one).
     subscription_id = event["data"]["object"]["id"]
     subscription = stripe.Subscription.retrieve(subscription_id, expand=["latest_invoice"])
-    donation_type = subscription["metadata"].get("donation_type", subscription["plan"]["metadata"].get("type", "membership"))
+    sub_meta = subscription["metadata"]
+
+    # When migrating existing recurring donations from salesforce to stripe subscriptions, we pass a field
+    # called "skip_sync" to the subscription metadata to let us know that we don't need to push anything back
+    # to salesforce. In that instance, we exit the function at this point.
+    if sub_meta.get("skip_sync", False):
+        return None
```

Comment:
**question:** If we're doing this in bulk, would it be at all helpful to log items we skip for debugging? Maybe we do that somewhere else so fine to ignore if so!

Reply:
Yeah, very helpful to log these but I'm including that in the command that sends recurring donation records to become subscriptions instead.